test/rendering/cases/map-worker/map.worker.js

```diff
@@ -0,0 +1,45 @@
+import Feature from '../../../../src/ol/Feature.js';
+import Map from '../../../../src/ol/Map.js';
+import View from '../../../../src/ol/View.js';
+import Point from '../../../../src/ol/geom/Point.js';
+import VectorLayer from '../../../../src/ol/layer/Vector.js';
+import VectorSource from '../../../../src/ol/source/Vector.js';
+
+/** @type {any} */
+const worker = self;
+
+worker.onmessage = (event) => {
+  if (event.data.action !== 'render') {
+    return;
+  }
+  const map = new Map({
+    pixelRatio: 1,
+    target: new OffscreenCanvas(256, 256),
+    layers: [
+      new VectorLayer({
+        source: new VectorSource({
+          features: [
+            new Feature({
+              geometry: new Point([0, 0]),
+            }),
+          ],
+        }),
+      }),
```

Comment:
To avoid regressions on this, please use a circle style with a larger radius and also a fill - this ensures that the rendered point is not below the pixel tolerance threshold of the image comparison of the rendering test.

Reply:
You were absolutely right, the test (which was basically the existing rendering test `map`), did not fail when rendering a blank image because of the tolerance threshold. I have updated the test to fail correctly, the `map` rendering test should also be updated at some point.